packages/styleguide/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@ontotext/styleguide",
+  "scripts": {
+    "start": "webpack serve",
+    "start:standalone": "webpack serve --env standalone",
+    "build": "rm -rf dist && npm run build:tokens && npm run build:webpack",
+    "build:tokens": "node src/index.mjs",
+    "build:webpack": "webpack --mode=production",
+    "analyze": "webpack --mode=production --env analyze",
+    "lint": "eslint src --ext js",
+    "format": "prettier --write .",
+    "check-format": "prettier --check .",
+    "test": "cross-env BABEL_ENV=test jest --passWithNoTests",
+    "watch-tests": "cross-env BABEL_ENV=test jest --watch",
+    "coverage": "cross-env BABEL_ENV=test jest --coverage"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.3",
+    "@babel/eslint-parser": "^7.23.3",
+    "@babel/plugin-transform-runtime": "^7.23.3",
+    "@babel/preset-env": "^7.23.3",
+    "@babel/runtime": "^7.23.3",
+    "babel-jest": "^27.5.1",
+    "concurrently": "^6.2.1",
+    "cross-env": "^7.0.3",
+    "eslint": "^7.32.0",
+    "eslint-config-important-stuff": "^1.1.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^3.4.1",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^27.5.1",
+    "jest-cli": "^27.5.1",
+    "prettier": "^2.8.8",
+    "pretty-quick": "^3.1.1",
+    "webpack": "^5.89.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-config-single-spa": "^7.0.0",
+    "webpack-merge": "^5.8.0"
+  },
+  "dependencies": {
+    "@tokens-studio/sd-transforms": "^1.3.0",
+    "@types/jest": "^27.0.1",
+    "@types/systemjs": "^6.1.1",
```

Comment:
This should not be needed

Reply:
Yes I used the cli.
I removed all unused deps.